### DISCUSSION TITLE
TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType'

### DISCUSSION THREAD

**Discussion by farkhandaathar** [mitmproxy/mitmproxy#5269] [General]
The code is 

    if (flow.server_conn and flow.server_conn not in SERVERS_SEEN):
        connect_time = (flow.server_conn.timestamp_tcp_setup -
                        flow.server_conn.timestamp_start)

        if (flow.server_conn.timestamp_tls_setup is not None):
            ssl_time = (flow.server_conn.timestamp_tls_setup -
                        flow.server_conn.timestamp_tcp_setup)

I am getting the following error. Can anyone help me in this matter?
Addon error: Traceback (most recent call last):
  File "mitmInterceptor.py", line 149, in response
    connect_time = (flow.server_conn.timestamp_tcp_setup -
TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType'

**Comment by mhils** [comment]
Please don't cross-post. https://stackoverflow.com/questions/71934483/typeerror-unsupported-operand-types-for-nonetype-and-nonetype-in-mitmp
